ynr/apps/people/models.py

```diff
@@ -703,26 +702,33 @@ def dict_for_csv(self, base_url=None):
         return row
 
     @property
-    def primary_image_model(self):
+    def person_image_model(self):
```

Comment:
I think this could be a `cached_property` so the below pattern of `if person.image / person.image.file` doesn't cause two DB requests?